src/drivers/batt_smbus/batt_smbus.cpp

```diff
@@ -552,37 +362,46 @@ BATT_SMBUS::cycle()
 
 			// calculate total discharged amount
 			new_report.discharged_mah = (float)((float)_batt_startup_capacity - (float)tmp);
-		}
 
-		// read battery temperature and covert to Celsius
-		if (read_reg(BATT_SMBUS_TEMP, tmp) == OK) {
-			new_report.temperature = (float)(((float)tmp / 10.0f) + CONSTANTS_ABSOLUTE_NULL_CELSIUS);
-		}
-
-		//Check if remaining % is out of range
-		if ((new_report.remaining > 1.00f) || (new_report.remaining <= 0.00f)) {
-			new_report.warning = battery_status_s::BATTERY_WARNING_EMERGENCY;
-		}
+			//Check if remaining % is out of range
+			if ((new_report.remaining > 1.00f) || (new_report.remaining <= 0.00f)) {
+				new_report.warning = battery_status_s::BATTERY_WARNING_EMERGENCY;
+				PX4_INFO("Percent out of range: %4.2f", (double)new_report.remaining);
+			}
 
-		//Check if discharged amount is greater than the starting capacity
-		else if (new_report.discharged_mah > (float)_batt_startup_capacity) {
-			new_report.warning = battery_status_s::BATTERY_WARNING_EMERGENCY;
-		}
+			//Check if discharged amount is greater than the starting capacity
+			else if (new_report.discharged_mah > (float)_batt_startup_capacity) {
+				new_report.warning = battery_status_s::BATTERY_WARNING_EMERGENCY;
+				PX4_INFO("Discharged greater than startup capacity: %4.2f", (double)new_report.discharged_mah);
+			}
 
-		// propagate warning state
-		else {
-			if (new_report.remaining > _low_thr) {
-				new_report.warning = battery_status_s::BATTERY_WARNING_NONE;
+			// propagate warning state
+			else {
```

Comment:
This is nitpicky, but can you put the brace on the same line?
` } else {`

Reply:
Fixed